ROMFS/px4fmu_common/init.d/airframes/4450_generic_quad_450

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# @name Generic Quadcopter (450mm)
+#
+# @type Quadrotor x
+# @class Copter
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set MC_ROLL_P 7.0
+	param set MC_ROLLRATE_P 0.15
```

Comment:
These are all default, can you drop them?

Reply:
I think @dagar 's original intent here was basically a placeholder.  I don't have a 450 airframe to know if the default values are appropriate.  Do we know of any other good candidate 450 sized airframes aside from the flame wheel that might need different parameter values?

For now, I'll add a commit that drops these values and also drops the default values from `4011_dji_f450`.  Thanks @bkueng !